src/Observables.jl

```diff
@@ -26,6 +26,11 @@ mutable struct Observable{T} <: AbstractObservable{T}
     Observable{T}(val) where {T} = new{T}([], val)
 end
 
+"""
+Alias of `Observable`.  Lika a `Ref`, but updates can be watched by adding a handler using [`on`](@ref).
```

Comment:
No reason to duplicate docstrings, makes it harder to maintain in the long run. "Alias of `Observable`." is enough IMO.